docs/user-guide/topics/ab_testing.md

```diff
@@ -0,0 +1,203 @@
+---
+template: main.html
+---
+
+# A/B/n Testing
+
+A/B/n testing relies on business metrics typically computed by a frontend, user-facing, application component. Metric values often depend on one or more interactions with backend (not user-facing) components. To A/B/n test a backend component, it is necessary to be able to associate a metric value (computed by the frontend) to the version of the backend component that contributed to its computation. The challenge is that the frontend service often does not know which version of the backend component processed a given request.
+To address this challenge, Iter8 introduces an A/B/n SDK which provides a frontend service with two APIs:
+
+a. **Lookup()** - Identifies a version of a backend component to send a request to. Given a user session, *Lookup()* returns a track identifier that can be used to route requests. So long as there are no changes in configuration, the track identifier (and hence route) will be same for the same user session, guaranteeing session stickiness. Note that the track identifier is not a version identifier; the version associated with a track changes over time as the versions being tested chanage.
+
+b. **WriteMetric()** - Associates a metric value with a backend component. Given a user session, *WriteMetric()* identifies the implementing version and writes the metric to a metric store associated with that version. By default, Iter8 uses a Kubernetes secret to store the metrics. The [abnmetrics](../tasks/abnmetrics.md) experiment task can be used to read the metrics stored by *WriteMetric()*.
+
+The Iter8 SDK is implemented using gRPC. The formal defintion of the API is in this [protocol buffer document](https://raw.githubusercontent.com/iter8-tools/docs/v0.13.1/samples/abn-sample/frontend/abn.proto).
+
+## Deploying the Implementing Service
```

Comment:
Could we please change the subtitle here to `Configuring the Iter8 A/B/n service`?

Reply:
It would be better if we use the terms `distributed application`, `frontend component`, and `backend component`, and use them consistently.
***
In the use-case we have presented, A/B/n service is watching for versions of the `backend component` which is part of a `distributed application` consisting of both a `frontend component` and `backend component`.